Python/SVY21.py

```diff
@@ -143,17 +148,50 @@ def computeLatLon(self, N, E):
         # Compute Latitude
         latFactor = tPrime / (self.k * rhoPrime)
         latTerm1 = latFactor * ((Eprime * x) / 2)
-        latTerm2 = latFactor * ((Eprime * x3) / 24) * ((-4 * psiPrime2) + (9 * psiPrime) * (1 - tPrime2) + (12 * tPrime2))
-        latTerm3 = latFactor * ((Eprime * x5) / 720) * ((8 * psiPrime4) * (11 - 24 * tPrime2) - (12 * psiPrime3) * (21 - 71 * tPrime2) + (15 * psiPrime2) * (15 - 98 * tPrime2 + 15 * tPrime4) + (180 * psiPrime) * (5 * tPrime2 - 3 * tPrime4) + 360 * tPrime4)
+        latTerm2 = latFactor * ((Eprime * x3) / 24) * (
+                (-4 * psiPrime2) + (9 * psiPrime) * (1 - tPrime2) + (12 * tPrime2))
+        latTerm3 = latFactor * ((Eprime * x5) / 720) * (
+                (8 * psiPrime4) * (11 - 24 * tPrime2) - (12 * psiPrime3) * (21 - 71 * tPrime2) + (
+                15 * psiPrime2) * (15 - 98 * tPrime2 + 15 * tPrime4) + (180 * psiPrime) * (
+                        5 * tPrime2 - 3 * tPrime4) + 360 * tPrime4)
         latTerm4 = latFactor * ((Eprime * x7) / 40320) * (1385 - 3633 * tPrime2 + 4095 * tPrime4 + 1575 * tPrime6)
         lat = latPrime - latTerm1 + latTerm2 - latTerm3 + latTerm4
 
         # Compute Longitude
         secLatPrime = 1. / math.cos(lat)
         lonTerm1 = x * secLatPrime
         lonTerm2 = ((x3 * secLatPrime) / 6) * (psiPrime + 2 * tPrime2)
-        lonTerm3 = ((x5 * secLatPrime) / 120) * ((-4 * psiPrime3) * (1 - 6 * tPrime2) + psiPrime2 * (9 - 68 * tPrime2) + 72 * psiPrime * tPrime2 + 24 * tPrime4)
+        lonTerm3 = ((x5 * secLatPrime) / 120) * ((-4 * psiPrime3) * (1 - 6 * tPrime2) + psiPrime2 * (
+                9 - 68 * tPrime2) + 72 * psiPrime * tPrime2 + 24 * tPrime4)
         lonTerm4 = ((x7 * secLatPrime) / 5040) * (61 + 662 * tPrime2 + 1320 * tPrime4 + 720 * tPrime6)
         lon = (self.oLon * math.pi / 180) + lonTerm1 - lonTerm2 + lonTerm3 - lonTerm4
 
+        #X IS LON AND Y IS LAT
```

Comment:
Oh, and this comment seems to be a little confusing now. Maybe it'll be better to shift this to the function below or just remove it.